.eslintrc.json

```diff
@@ -1,7 +1,9 @@
 {
     "env": {
         "browser": true,
-        "es2021": true
+        "es2021": "When true, a modal dialog will show if the chosen local port isn't used for forwarding.",
```

Comment:
What does this change mean? Is this even a legal value?

Reply:
Hahaha, you know what this is? I bet it is a GitHub Copilot artifact. It probably made a suggestion and I hit Tab without noticing. 🤣

Reply:
So I guess this will be fixed too? ;)

Reply:
I thought I fixed it, but it seems I forgot... Should be OK now!